src/game_api/state.cpp

```diff
@@ -539,7 +539,7 @@ void State::set_seed(uint32_t seed)
 SaveData* State::savedata()
 {
     auto gm = get_game_manager();
-    return gm->save_related->savedata.decode();
+    return gm->save_related->savedata.decode(); // wondering if it matters if it's local or not?

```

Comment:
It probably kind of is? if someone unlocks something in online and then immediately gets re-sync to earlier moment

Reply:
I guess using the local state is always a good thing to do

Reply:
I think i won't change this thou, it's not super important. Maybe you can add that to your online PR stuff